src/aitsi/m3spin/pkb/exception/SimplePkbException.java

```diff
@@ -0,0 +1,7 @@
+package aitsi.m3spin.pkb.exception;
+
+public class SimplePkbException extends Exception{
```

Comment:
Ja to "Simple" dałem w nazwie SimpleParserException dlatego, że to parser języka SIMPLE, a nie dlatego, że to prosty wyjątek XD. Zmienić na PkbException albo PKBException (zależnie od przyjętej konwencji).